app/views/schedules/_schedule_library_visit_dropdown.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="schedule-once-dropdown dropdown">
+  <button class="btn btn-primary dropdown-toggle" type="button" id="scheduleOnceDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    🗓 Schedule access to ...
```

Comment:
@jvine the 🗓 wasn't in the mockup on #557, but I added it here. Let me know if I should remove it.

Reply:
Incidentally, that was probably an artifact from my functional mock (using the old ScheduleOnce embed). @jvine's mocks had another icon.

Reply:
Oops yes, it's appropriate to add whatever calendar icon we are using to that button. Missed it in my mock, sorry.